docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -88,26 +99,26 @@ We will start by creating files for our project and deleting the default files t
 First, delete the old files. Run:
 
 ```sh
-rm src/Add.ts
-rm src/Add.test.ts
-rm src/interact.ts
+$ rm src/Add.ts
+$ rm src/Add.test.ts
+$ rm src/interact.ts
 ```
 
 And generate the new files for our project. Enter:
 
 ```sh
-zk file src/Square
-touch src/main.ts
+$ zk file src/Square
+$ touch src/main.ts
 ```
 
 The `zk file <name>` command created both `src/Square.ts` and `src/Square.test.ts` for us, but we won't worry about writing tests in this tutorial. We'll use `main.ts` as a script to interact with the smart contract and observe how it works. In later tutorials, we will see how to interact with a smart contract from the browser, like a typical end user, but for now we'll just use our `main.ts` script to do this.
 
 Now, let's open `src/index.ts` in a text editor and change it to look like below. This file contains all exports we want to make available for consumption from outside our smart contract project, such as from a UI.
 
-```
-import { Square } from './Square.js';
-
-export { Square };
+```ts src/index.ts
```

Comment:
This syntax is valid per the CommonMark spec. Docusaurus expects CommonMark markdown, and thus syntax highlighting continues to work great in the docs with this syntax.

The string after the triple backticks but before the newline is called an "info string" in the spec. See https://spec.commonmark.org/0.30/#info-string for more… info 😅